lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -22,12 +25,14 @@
 ]
 
 
-def get_problem_grade_distribution(course_id):
+def get_problem_grade_distribution(course_id, enrollment):
     """
     Returns the grade distribution per problem for the course
 
     `course_id` the course ID for the course interested in
 
```

Comment:
The rest of the file follows the empty line between arguments convention, but it's easy enough to switch everywhere if we want.  I was erring on the side of keeping everything the same and not introducing extra changes.